### DISCUSSION TITLE
Position the `cmdline` at cursor position? (feature request)

### DISCUSSION THREAD

**Discussion by molleweide** [folke/noice.nvim#327] [Q&A] [answered]
Will there be support for opening the cmdline/search at cursor position similar to how one can use the cursor theme with telescope?

I think that this would be a great feature from an ergonomic point of views since it would reduce jerk neck movements when moving eye focus to the cmdline. 

After a quick browse through the docs I concluded that cursor position feature did not exist but I am still not sure, so forgive me if I did not research the issue enough.

**Comment by folke** [comment, marked as answer]
You can do this with config:

```lua
    cmdline = {
      opts = {
        relative = "cursor",
        position = { row = -2, col = 0 },
      },
    },
```

Make sure to disable the `command_palette` preset

**Reply by molleweide** [reply]
Thank you very much Folke!
